README.rst

```diff
@@ -26,6 +26,14 @@ then run the following command to install the other packages that the draft requ
 
    sudo tlmgr install latexmk isodate substr relsize ulem fixme rsfs extract layouts enumitem
 
+-------------------------
+Getting Started on Ubuntu
```

Comment:
It could. I picked Ubuntu because that's what OS I experienced the issue on. I suppose it could say "Getting Started on Systems With apt-get"?